community/kernel/src/test/java/org/neo4j/test/subprocess/SubProcess.java

```diff
@@ -214,7 +197,7 @@ private String classPath( String parentClasspath )
 
     private static Process start( String... args )
     {
-        ProcessBuilder builder = new ProcessBuilder( args );
+        ProcessBuilder builder = new ProcessBuilder( args ).inheritIO();
```

Comment:
@chrisvest always inheriting the IO streams seems like the wrong thing to do IMO.